src/firebase/notifications.ts

```diff
@@ -57,12 +57,12 @@ export function* handleNotification(
   notificationState: NotificationReceiveState
 ) {
   ValoraAnalytics.track(AppEvents.push_notification_opened, {
-    id: message.data?.id,
+    id: message.data?.id.toString(),
```

Comment:
there was a change to the type declarations where every key in `data` can be a string or object, but it should not affect these particular properties `https://github.com/invertase/react-native-firebase/pull/7316/files`